crash_similarity.py

```diff
@@ -2,17 +2,20 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import bisect
+import json
+import multiprocessing
 import os
-import time
-import download_data
-import utils
 import random
+import time
+
 import gensim
 import smart_open
```

Comment:
We can move the usage of `smart_open` in utils.py, this way we can avoid the `open_file_function` argument for `read_files_generator`.
I don't expect to use `smart_open` other than in utils.py, so it should be fine to move it there.